assets/app/view/game/spreadsheet.rb

```diff
@@ -104,18 +103,18 @@ def render_title
         [
           h(:tr, [
             h(:th, ''),
-            h(:th, { attrs: { colspan: @game.players.size } }, 'Players'),
-            h(:th, { attrs: { colspan: 2 } }, 'Bank'),
-            h(:th, { attrs: { colspan: 2 } }, 'Prices'),
-            h(:th, { attrs: { colspan: 4 } }, 'Corporation'),
+            h(:th, spacing[@game.players.size], 'Players'),
```

Comment:
shouldn't you be calling the lambda with parens?

Reply:
You tell me. I don’t mind changing these to `spacing.call(x)`.